packages/salesforcedx-vscode-soql/package.json

```diff
@@ -21,10 +21,10 @@
   ],
   "dependencies": {
     "@salesforce/core": "^2.12.1",
-    "@salesforce/soql-builder-ui": "0.0.10",
-    "@salesforce/soql-data-view": "0.0.5",
-    "@salesforce/soql-language-server": "0.2.1",
-    "@salesforce/soql-tmlanguage": "0.1.1",
+    "@salesforce/soql-builder-ui": "^0.0.20",
+    "@salesforce/soql-data-view": "^0.0.5",
+    "@salesforce/soql-language-server": "^0.2.5",
+    "@salesforce/soql-tmlanguage": "^0.1.1",
```

Comment:
@jgellin-sf I've got these dependencies locked down.  And updated to the latest.  Can you match and remove the ^?

https://github.com/forcedotcom/salesforcedx-vscode/pull/2695/files

Reply:
@dehru As you wish. See `chore(soql): lock down dependency versions`.

Reply:
@dehru CircleCI is complaining about `@salesforce/soql-language-server 0.2.7`. Is this version not published yet?